samza-core/src/main/java/org/apache/samza/util/DiagnosticsUtil.java

```diff
@@ -122,6 +122,8 @@ public MetadataFileContents(String version, String metricsSnapshot) {
       DiagnosticsManager diagnosticsManager = new DiagnosticsManager(jobName, jobId, containerId, execEnvContainerId.orElse(""), taskClassVersion,
           samzaVersion, hostName, diagnosticsSystemStream, systemProducer, Duration.ofMillis(new TaskConfig(config).getShutdownMs()));
 
+      Option<String> blacklist = ScalaJavaUtil.JavaOptionals$.MODULE$.toRichOptional(
```

Comment:
This can just be 
ScalaJavaUtil.JavaOptionals.toRichOptional(metricsConfig.getMetricsSnapshotReporterBlacklist(diagnosticsReporterName)).toOption()
right?

Reply:
That's what I tried initially, but the IDE complains that it can't find `JavaOptionals` when doing that. Looks like this `MODULE$` syntax is used in some other places too.

Reply:
can this helper that converts `optional`s between java<->scala be added to one of our util classes?